### DISCUSSION TITLE
First time install - but what do I say is the Ingame-name ?

### DISCUSSION THREAD

**Discussion by KevinFromUpThere** [gnmyt/MCDash#129] [Q&A]
Followed the instructions, changed the password, but no documentation mentions the Ingame-name - and when i try putting my MC username I just get "Login failed, check your credentials"?
![image](https://github.com/gnmyt/MCDash/assets/8724155/239254c8-7d9e-4b58-a767-e660d0044ce6)

The INSTALLATION instructions say how to change the password through the console but don't mention anything about the ingame-name
https://mcdash.gnmyt.dev/docs/intro/

**Comment by Crosis47** [comment]
For anyone else looking for this info, the username would be "CONSOLE" and it is case-sensitive
